fedimint-server/src/config.rs

```diff
@@ -249,6 +250,11 @@ impl ServerConfig {
         let client_config = ClientConfig {
             federation_name: peer0.federation_name.clone(),
             nodes: peer0.api.nodes("ws://", names),
+            epoch_pk: server_config
+                .get(&peers[0])
+                .expect("must have at least one peer")
+                .epoch_pk_set
+                .public_key(),
```

Comment:
Didn't we just remove the custom client config creation code? Seems we forgot a few places @jkitman ?

Reply:
Still redundant code that we have to maintain, reopened the issue.